src/DeuxEtageres.cpp

```diff
@@ -160,21 +177,22 @@ void DeuxEtageres::step() {
             
             float sumout = lpout*lpgain + hpout*hpgain +  bp2out*bp2gain + bp3out*bp3gain;
 
-            outputs[OUTL_OUTPUT + i].value = sumout;
+            outputs[OUTL_OUTPUT + i].setVoltage(sumout);
     
             // Lights
             lights[CLIPL_LIGHT + i].setBrightnessSmooth( fabs(sumout) > 10. ? 1. : 0. );    
```

Comment:
This is technically deprecated but it wasn't clear how to migrate to the new method (`setSmoothBrightness` or some such)